examples/Cargo.toml

```diff
@@ -7,7 +7,7 @@ version = "0.0.0"
 [dev-dependencies]
 anyhow = "1.0.33"
 futures = "0.3.6"
-openssl = "0.10.32"
+openssl = "0.10"
```

Comment:
Nit: is this needed?

Reply:
nope, sneaked in, I'll drop it (we should apply it either way, but in a separate patch)